### PR TITLE
/who [username] pulls down basic information from gravatar.com #76

### PR DESCRIPTION
**Pull request by nschonni**
Extends the /who command with a user's Gravatar information. There is a Gravatar API limitation where if the email used is not the primary address on the Gravatar account the profile may not show https://secure.gravatar.com/site/implement/profiles/.

### CODE REVIEW

**Comment by davidfowl** [comment]
Can you update this one?

File: JabbR/Chat.js [hunk 1]
```diff
@@ -452,6 +452,13 @@
             ui.addMessage('Note: ' + userInfo.Note, 'list-item');
         }
 
+      $.getJSON('https://secure.gravatar.com/' + userInfo.Hash + '.json?callback=?', function (profile) {
+        ui.addMessage('Gravatar', 'list-header');
```
Comment (davidfowl) [inline JabbR/Chat.js]: Any xss issues here?

Comment (nschonni) [inline JabbR/Chat.js]: I don't know of any, are you talking about malicious content in the Gravatar profile return? The only parameter (Hash) is generated by JabbR and not user input.
The only one I wouldn't be 100% on is the returned value from the Name portion of the response as that could be set by the user.

Comment (samandmoore) [inline JabbR/Chat.js]: So can You test it out with malicious content? And trying to encode it or something?

File: JabbR/Chat.js [hunk 2]
```diff
@@ -452,6 +452,10 @@
             ui.addMessage('Note: ' + userInfo.Note, 'list-item');
         }
 
+        $.getJSON('https://secure.gravatar.com/' + userInfo.Hash + '.json?callback=?', function (profile) {
+          ui.showGravatarProfile(profile.entry[0]);
```
Comment (davidfowl) [inline JabbR/Chat.js]: 4 spaces, no tabs.

**Comment by nschonni** [comment]
Something got messed up when I merged with the upstream. Will fix and submit a new PR with a clean push.

**Pull request closed without merging**